packages/web-components/src/components/ic-radio-group/ic-radio-group.tsx

```diff
@@ -42,6 +42,11 @@ export class RadioGroup {
    */
   @Prop() required: boolean = false;
 
+  /**
+   * The orientation of the radio buttons in the radio group. If there are more than two radio buttons in a radio group, then the orientation of the radio will always be vertical.
+   */
+  @Prop() orientation: IcOrientation = "vertical";
```

Comment:
Does this need to be mutable to prevent warnings as it's potentially gets modified inside the component code?